islandora_transforms/LICENSE.txt

```diff
@@ -0,0 +1 @@
+{"error":"Not Found"}
```

Comment:
Wat's up with this?

Reply:
looks like it's busted in core repo.. all the forks have this.